Documentation/infra-guide/arm64-infra.md

```diff
@@ -0,0 +1,56 @@
+# etcd arm64 test infrastructure
+
+## Infrastructure summary
+
+All etcd project pipelines run via github actions. The etcd project currently maintains dedicated infrastructure for running `arm64` continuous integration testing. This is required because currently github actions runner virtual machines are only offered as `x64`.
+
+The infrastructure consists of two `c3.large.arm` bare metal servers kindly provided by [Equinix Metal](https://www.equinix.com/) via the [CNCF Community Infrastructure Lab](https://github.com/cncf/cluster/issues/227).
+
+| Hostname                      | IP             | Operating System   | Region        |
+|-------------------------------|----------------|--------------------|---------------|
+| etcd-c3-large-arm64-runner-01 | 86.109.7.233   | Ubuntu 22.04.1 LTS | Washington DC |
+| etcd-c3-large-arm64-runner-02 | 147.28.151.226 | Ubuntu 22.04.1 LTS | Washington DC |
+
+## Granting infrastructure access
+
+Etcd arm64 test infrastructure access is closely controlled to ensure the infrastructure is secure and protect the integrity of the etcd project.
+
+Access to the infrastructure is defined by the infra admins table below:
+
+| Name                      | Github         | K8s Slack          | Email              |
```

Comment:
Can we make it a separate file in some format that could be used for automation? There is no benefit of displaying this here to readers of this doc.

Reply:
I have no strong opinion here. Ideally, moving the list to a separate file and automating the process (e.g. grant access or revoke access) is good, but we will most likely not update the list very often. So putting the list here is simple and good enough for now, and we can update it in future when needed.

Reply:
I think initially let's get these basic controls merged so I can start implementation, I'm keen to continue contributing in this area and iterating on this in future.